layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c

```diff
@@ -1,67 +1,81 @@
-   * │Vol-│Rwd │F FD│           Play         │Stp│   │Prv│P D│Nxt│
-   * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┘
-   */
+  /* Fn Layer [1] */
 	 [1] = LAYOUT_60_ansi_arrow(KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL,
                                 KC_BRIU, KC_TRNS, KC_TRNS, KC_END, RESET, KC_TRNS, KC_TRNS, KC_TRNS, KC_INS, KC_TRNS, KC_PSCR, KC_TRNS, KC_TRNS, KC_EJCT,
                                 KC_BRID, KC_TRNS, KC_SLCK, KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MUTE,
                                 KC_VOLU, KC_TRNS, KC_TRNS, KC_CALC, KC_TRNS, KC_TRNS, KC_NLCK, KC_MAIL, KC_TRNS, KC_TRNS, KC_PAUS, KC_PGUP, KC_TRNS,
                                 KC_VOLD, KC_MRWD, KC_MFFD, KC_MPLY, KC_MSTP, KC_TRNS, KC_MPRV, KC_PGDN, KC_MNXT
      ),
-
-  /*
-   * Fn Layer [2]
-   * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
-   * │   │   │   │   │   │   │   │   │   │   │   │   │   │       │
-   * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
-   * │     │R T│R M│Hu+│Hu-│Sa+│Sa-│Br+│Br-│   │   │   │   │     │
-   * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
-   * │      │   │   │   │   │   │   │   │   │   │   │   │        │
-   * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┬───┤
-   * │        │   │   │   │BL-│B T│BL+│B C│   │   │      │   │   │
-   * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬───┼───┼───┤
-   * │    │    │    │                        │   │   │   │   │   │
-   * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┘
-   */
+  /* Fn Layer [2] */
 	 [2] = LAYOUT_60_ansi_arrow(KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
                                 KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
                                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
                                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, BL_DEC, BL_TOGG, BL_INC, BL_STEP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
                                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
      )
 };
+
+void keyboard_post_init_user(void) {
+  rgblight_enable_noeeprom();
+  rgblight_sethsv_noeeprom(0, 0, 0);
+}
+
+/* RGB Lighting/LED/Underglow as Caps Lock, Num Lock, Scroll Lock, and Layer Indicator */
+   void update_led(void) {
+     /* Num Lock Indicator */
+        if (host_keyboard_led_state().num_lock) {
+          rgblight_setrgb(225, 8, 0);
+        }
+     /* Scroll Lock Indicator */
+        if (host_keyboard_led_state().scroll_lock) {
+          rgblight_setrgb(241, 190, 72);
+        }
+   }
+
+   void led_set_user(uint8_t usb_led) {
```

Comment:
This should be updated to use:
https://docs.qmk.fm/#/custom_quantum_functions?id=led-control


Also, it looks/sounds like you may want to look into the rgb lighting layers stuff.
https://docs.qmk.fm/#/feature_rgblight?id=lighting-layers

Reply:
Can you suggested the change @drashna?